sormas-api/src/main/java/de/symeda/sormas/api/sample/SampleExportDto.java

```diff
@@ -27,8 +32,7 @@
 	private String uuid;
 	private String labSampleID;
 	private String epidNumber;
-	private String firstName;
-	private String lastName;
+	private AssociatedCase associatedCase;
```

Comment:
It is necessary because after implementing export samples with contacts the fistName and lastName would come from the contact person and it would be a bit messy to have case and contact data in separate objects but their person name in common fields

Reply:
Okay, I'm fine leaving it like this for now.